src/conn/conn_layered.c

```diff
@@ -1831,45 +1830,71 @@ __layered_drain_ingest_tables(WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __layered_update_prune_timestamps_print_update_logs --
+ *     Print logs for the prune timestamp update.
+ */
+static void
```

Comment:
Can this be inline? We can add WT_INLINE

Reply:
that should be inline, thanks! let me fix it :)